geolocator/lib/geolocator.dart

```diff
@@ -8,7 +8,7 @@ import 'package:geolocator_platform_interface/geolocator_platform_interface.dart
 export 'package:geolocator_platform_interface/geolocator_platform_interface.dart';
 export 'package:geolocator_android/geolocator_android.dart'
     show AndroidSettings;
-export 'package:geolocator_apple/geolocator_apple.dart' show AppleSettings;
+export 'package:geolocator_apple/geolocator_apple.dart';
```

Comment:
The `ActivityType` should indeed be exported. However it would be better to add it to the `show` condition. 

We are in the progress of adding more logic to the `geolocator_apple` package and we don't want to expose everything by default through the `geolocator` app-facing package.

```suggestion
export 'package:geolocator_apple/geolocator_apple.dart' 
         show AppleSettings, ActivityType;
```

Reply:
> 

Yes. I fixed it!